### PR TITLE
OCPBUGS#30924: Node reboot warning

### PR DESCRIPTION
**Pull request by snarayan-redhat**
Version(s): 4.12+

Issue: https://issues.redhat.com/browse/OCPBUGS-30924

Link to docs preview:

QE review:
- [x] QE has approved this change.

### CODE REVIEW

File: modules/nw-proxy-configure-object.adoc [hunk 1]
```diff
@@ -29,6 +29,11 @@ A cluster administrator can configure the proxy for {product-title} by modifying
 Only the `Proxy` object named `cluster` is supported, and no additional proxies can be created.
 ====
 
+[WARNING]
+====
+The Machine Config Operator (MCO) triggers node reboot when the cluster-wide proxy is enabled.
```
Comment (bhardesty) [inline modules/nw-proxy-configure-object.adoc:34]: Can you be more specific? Does it cause all nodes to reboot or only certain ones? Is there anything the user should avoid doing while the nodes are rebooting? Maybe say something like this (which you might need to adjust to be technically accurate):

```suggestion
Enabling the cluster-wide proxy will trigger all non-control plane nodes to reboot one at a time. Make this change outside of peak usage hours for your cluster and avoid upgrading or hibernating your cluster until all nodes have rebooted.
```

**Review by bhardesty — COMMENTED** [review]
Looks good! I just made a suggestion to make the added admonition more clear.

File: modules/nw-proxy-configure-object.adoc (hunk 1 above)
Comment (snarayan-redhat) [reply modules/nw-proxy-configure-object.adoc:34]: Good thought, Ben! IMO, we try not to be overly prescriptive of what they should be doing. The user needs to know that the reboot would happen and let them take call to deal with it.

**Pull request merged**